corpora-i-ching/shift_icons.py

```diff
@@ -0,0 +1,36 @@
+from PIL import Image
+import os
+
+# Configuration
+SHIFT_PIXELS = -1
```

Comment:
one-off that can moves now 1 pixels to the left .. could be deleted, cool script tho ..